tests/unit/projection-test.js

```diff
@@ -711,30 +718,21 @@ module('unit/projection', function(hooks) {
                 age: AUTHOR_AGE,
               },
             },
-            meta: {
-              projectionTypes: [BOOK_CLASS_PATH]
```

Comment:
This still has value because when it is present it alerts us that this data is partial data, we otherwise do not have a way of determining this. I was planning on using it's presence for developing the `load-data-but-leave-in-unloaded-state` behavior.

Reply:
Should we just have a flag `partial: true`?

Reply:
Might be easier yes